spec/hugging_face/inference_api_spec.rb

```diff
@@ -82,6 +82,28 @@
     end
   end
 
+  describe '#sentiment' do
+    let(:text)  { 'Text to analyse' }
+    let(:input) { { inputs: text } }
+    let(:output) { [{"label"=>"NEGATIVE", "score"=>0.999495267868042}, {"label"=>"POSITIVE", "score"=>0.0005046788137406111}].to_json }
+
+    context 'with no model provided'  do
+      let(:url) { "https://api-inference.huggingface.co/models/sshleifer/distilbart-xsum-12-6" }
+
+      subject { api.summarization(input: text) }
```

Comment:
Commented in another PR about this. Should be `sentiment`. 

Probably need to figure out a way to properly mock hugging face API, so that if you call the wrong method in tests it would actually fail.

Reply:
Ah - well spotted, now fixed, see two commits below

And yes agreed.